src/pages/index.js

```diff
@@ -297,6 +297,9 @@ const Page = () => (
             <IndexCardLink to="/partner_docs/ThalesGuide">
```

Comment:
why do we have Thales info in this?  also on next line.

Reply:
This is a global file that displays on the main page and lists all the partners. For each new partner guide, the file has to updated to add an entry for the new guide